src/popup/pages/home.tsx

```diff
@@ -137,10 +138,12 @@ const Home = () => {
                         </Swiper>
                     )}
 
+                    {Boolean(toolsRef.current?.children.length) && <h3 className="text font-medium text-base leading-10">Tools:</h3>}
```

Comment:
What is this `Boolean`? Are you casting type on the result? Do you think this is needed?

Reply:
The `&&` will cast it to a Boolean implicitly, The `Boolean(...)` just makes it explicit.